walkgen_params:
  params:
    N_phase_return: 3 # Number of step to return (N_phase_return surfaces for each foot)
    com: False # Optimisation of the CoM
    horizon: 8 # Number of fstep opitimised.
    # order matters! LF LH RF RH
    contact_names: ["FL_foot","RL_foot","FR_foot","RR_foot"]
    shoulder_offsets: [[0.367, 0.2],[-0.367, 0.2],[0.367, -0.2],[-0.367, -0.2]] 

  heightmap:
    fitsize_x: 10
    fitsize_y: 5
    fitlength: 0.3 # half-square size centered around position of the robot.
    recompute_slope: False # Recompute the slope of the terrain for each contact phase configuration.